cogs/tmpc.py

```diff
@@ -391,6 +391,39 @@ async def kick(self, ctx: Context, member: User):
                              description=f"{member.mention} was kicked.")
         await ctx.reply(embed=embed)
 
+    @tmpc.command(pass_context=True,
+                  aliases=["remove", "rem", "rm"],
+                  brief="Deletes the tmpc even if someone is still in the channel",
+                  help="You can force delete this tmpc even if people are still in the voice channel.")
+    async def delete(self, ctx: Context):
+        """
+        Deletes the tmpc even if the voice channel is not empty
+
+        Args:
+           ctx: The command context provided by the discord.py wrapper.
+        """
+        document = await self.check_tmpc_channel(ctx)
+        try:
+            await document.voice.delete(reason="Force deleted by owner")
+        except NotFound:
+            pass
+        try:
+            await document.chat.delete(reason="Force deleted by owner")
+        except NotFound:
+            pass
+
+        if type(document) == StudyChannel:
+            for message in document.messages:
+                try:
+                    await message.delete()
+                except NotFound:
+                    pass
+            await self.study_db.delete_one({DBKeyWrapperEnum.ID.value: document._id})
+        else:
+            await self.gaming_db.delete_one({DBKeyWrapperEnum.ID.value: document._id})
+
+        logger.info(f"Force deleted Tmp Channel {document.voice.name}")
```

Comment:
Deleting non-existent channels will result in an AttributeError.
Use a generic log message instead.